mkt/prices/models.py

```diff
@@ -347,7 +347,13 @@ def create_addon_purchase(sender, instance, **kw):
                      c=instance))
         return
 
-    if instance.type == mkt.CONTRIB_PURCHASE:
+    print
+    print dir(instance.price_tier)
+    print instance.price_tier.price
+    print instance.price_tier.name
+
+    if instance.type == mkt.CONTRIB_PURCHASE \
```

Comment:
This bit is wrong at the moment as this should be run. The problem is the user profile bit isn't working as the user object here is not right.